Sources/Vapor/Request/Request.swift

```diff
@@ -309,6 +312,7 @@ public final class Request: CustomStringConvertible, Sendable {
         byteBufferAllocator: ByteBufferAllocator = ByteBufferAllocator(),
         on eventLoop: EventLoop
     ) {
+        let requestId = headers[.xRequestId].first ?? UUID().uuidString
```

Comment:
Looks fine but i'd prefer `.first(name:)`, will be slightly more performant as it wouldn't be filter/collecting stuff when eventually we do `.first` anyway:
```suggestion
let requestId = headers.first(name: .xRequestId) ?? UUID().uuidString
```